bin/sozo/src/ops/migration/mod.rs

```diff
@@ -486,7 +486,9 @@ where
 
     TransactionWaiter::new(transaction_hash, migrator.provider()).await?;
 
-    ui.print_sub(format!("Registered at: {transaction_hash:#x}"));
+    ui.print_header(format!("======================================================"));
```

Comment:
Also here it should not be a header as it should still under models. Perhaps only having an empty line to make the separation is good, or at least remove header.

Reply:
I thought adding some sort of divider would make the delineation that much clearer. I will remove it if it doesn't.